src/dex-ui/store/walletSlice/walletSlice.ts

```diff
@@ -23,12 +23,12 @@ const initialWalletState: WalletState = {
 const createWalletSlice: WalletSlice = (set, get): WalletStore => {
   return {
     ...initialWalletState,
-    getTokenAmountWithPrecision: (tokenSymbol: string, tokenAmount: number) => {
+    getTokenAmountWithPrecision: (tokenSymbol: string, tokenAmount: number, tokenId?: string) => {
```

Comment:
This is a temporary fix to get the token amount to precision based on the tokenId.